tests/test.yml

```diff
@@ -1,18 +1,32 @@
 ---
 
-- hosts: localhost
+- hosts: controller
   remote_user: root
+  become: True
+  tasks:
+    - name: Installing repo for Java 8 in Ubuntu
```

Comment:
Agreed, but I think that playbook should include the required PPA in trusty... we can do a PR or a fork, but it's ok to use it like this for now.